tests/test_maintain_resource_path.py

```diff
@@ -67,7 +70,7 @@ def test_create_request_without_username(client):
     assert request_id, "POST /request did not return a request_id"
     assert request_data == {
         "request_id": request_id,
-        "username": "requestor-user",  # username from access_token_patcher
+        "username": "requestor_user",  # username from access_token_patcher
```

Comment:
I was just wondering how did `requestor-user` work all these days?

Reply:
not all tests use the access_token_patcher, and those that do used the username from access_token_patcher, so as long as they match it's all good